pkg_defs/aca_hi_bgd/meta.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: aca_hi_bgd
+  version:  {{ SKA_PKG_VERSION }}
+
+build:
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+source:
+  path: {{ SKA_TOP_SRC_DIR }}/aca_hi_bgd
+
+# the build and runtime requirements. Dependencies of these requirements
+# are included automatically.
+requirements:
+  # Packages required to build the package. python and numpy must be
+  # listed explicitly if they are required.
+  build:
+    - python
+    - setuptools
+    - setuptools_scm
+    - setuptools_scm_git_archive
+
+  # Packages required to run the package. These are the dependencies that
+  # will be installed automatically whenever the package is installed.
+  run:
+    - python
+    - numpy
+    - astropy
+    - jinja2
+    - chandra.time
+    - pyyaks
+    - ska.engarchive
+    - kadi
+    - ska.matplotlib
+    - ska.numpy
+    - acdc
+    - testr
+
+test:
+  imports:
+    - aca_hi_bgd
+
+
+about:
+  home: git@github.com:sot/aca_hi_bgd
```

Comment:
Are we doing https now by default?  I thought so, but am not sure.

Reply:
I thought for the build script, the default is https on Windows and ssh on other, but I also thought that for the recipes it is somewhat moot.

Reply:
If it is moot then https is preferred since it makes `home` correspond to the actual definition of the `home` tag (a project URL that you can load in a browser) instead of our overloaded meaning of git-specific URL that our build script can use to get the repo.

Reply:
My moot was more along the lines of.. I figured they'd get a bulk edit at some point.  But ok.

Reply:
I also prefer https. We have had this conversation a few times. The build script can always be fixed. Actually... for packages with documentation, I would prefer to have:
- home: https://sot.github.io/aca_hi_bgd
- dev_url: https://github.com/sot/aca_hi_bgd
- doc_url: https://sot.github.io/aca_hi_bgd
but either way is fine. I suppose one day I will go and change all :-)

In previous iterations, Jean has said something about private repositories and ssh not working for her. I have never understood exactly.

Reply:
For me I think it was https not working. When we tried to swap to https last time, I think it asked for my credentials once for every private repo.  And from the bits of conversation between you two, I thought the https authentication was perhaps *understood* less than the ssh process?  But if it is working for the automated builds I don't care.

Reply:
the thing is... that credential request comes from our build script checking out the code, so we can easily replace all instances of https by ssh within the build script. The script already does that, it either uses all ssh or all https ([see here](https://github.com/sot/skare3/blob/master/ska_builder.py#L82)), so changing what is written in meta.yaml changes nothing.

Reply:
(or am I missing something?)

Reply:
I think that's where I said it was moot and Tom said that he'd then just prefer I used the https form in new meta.yaml recipes then. That's fine.